clang/test/dpct/use-pure-sycl-queue/main.cu

```diff
@@ -0,0 +1,116 @@
+// UNSUPPORTED: system-windows
+
+// RUN: cp %S/* .
+// RUN: dpct --use-pure-sycl-queue -p=%S -out-root %T --cuda-include-path="%cuda-path/include"
+// RUN: FileCheck --input-file %T/kernel1.dp.cpp --match-full-lines %S/kernel1.cu
+// RUN: FileCheck --input-file %T/kernel2.dp.cpp --match-full-lines %S/kernel2.cu
+// RUN: FileCheck --input-file %T/main.dp.cpp --match-full-lines %S/main.cu
+
+// RUN: grep -x "extern sycl::device dev_ct1;" %T/*.cpp | wc -l > %T/extern_count1.txt
+// RUN: FileCheck --input-file %T/extern_count1.txt --match-full-lines extern_count.txt
+// RUN: grep -x "extern sycl::queue q_ct1;" %T/*.cpp | wc -l > %T/extern_count2.txt
+// RUN: FileCheck --input-file %T/extern_count2.txt --match-full-lines extern_count.txt
+// RUN: grep -x "sycl::device dev_ct1;" %T/*.cpp | wc -l > %T/define_count1.txt
```

Comment:
how about use a "defualt_selector" for device.

Reply:
fixed